processing file asmFiles/RAW.asm (1 of 1)
processing file asmFiles/RAW.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 3)
--- memsim.hex	2021-10-24 04:07:00.517314367 -0400
+++ memcpu.hex	2021-10-24 04:07:00.396309987 -0400
@@ -81,38 +81,38 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
+:0400C1000000005AE1
+:0400C20000000051E9
+:0400C3000000003306
+:0400C400000000191F
+:0400C50000000050E7
+:0400C600000000290D
+:0400C700000000161F
+:0400C800000000151F
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
 :0400E1000000000714
 :0400E2000000000713
 :0400E3000000000712
@@ -145,69 +145,7 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:040C40000000037736
 :043FFE0000000500BA
+:043FFF00000000209E
 :00000001FF
processing file asmFiles/mult.asm (2 of 3)
--- memsim.hex	2021-10-24 04:07:03.181410797 -0400
+++ memcpu.hex	2021-10-24 04:07:03.123408697 -0400
@@ -1,3 +1,4 @@
+:04000000341DFFFCB0
 :040001008FA20000CA
 :0400020027BD000412
 :040003008FA30000C7
@@ -17,4 +18,5 @@
 :0400110027BDFFFC0C
 :04001200AFA6000095
 :04001300FFFFFFFFED
+:040C400000000002AE
 :00000001FF
processing file asmFiles/mult_procedure.asm (3 of 3)
--- memsim.hex	2021-10-24 04:07:05.590497996 -0400
+++ memcpu.hex	2021-10-24 04:07:05.501494774 -0400
@@ -30,7 +30,7 @@
 :04001D0027BDFFFC00
 :04001E00AFA6000089
 :04001F0003E00008F2
-:043FFC0000000010B1
-:043FFD00000000A020
-:043FFE00000013604C
+:040C400000000001AF
+:043FFD0000000010B0
+:043FFE000000000AB5
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2021-10-24 04:07:12.174736318 -0400
+++ memcpu.hex	2021-10-24 04:07:12.042731540 -0400
@@ -12,27 +12,5 @@
 :04000B0015E0FFF904
 :04000C00FFFFFFFFF4
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/search.asm (1 of 1)
--- memsim.hex	2021-10-24 04:07:19.873014964 -0400
+++ memcpu.hex	2021-10-24 04:07:19.795012141 -0400
@@ -14,8 +14,6 @@
 :04000D001000FFF9E7
 :04000E00AFA500009A
 :04000F00FFFFFFFFF1
-:0400200000000124B7
-:0400210000005C6F10
 :040022000000006476
 :040023000000087D54
 :0400240000005FCBAE
@@ -117,4 +115,5 @@
 :040084000000A4DBF9
 :0400850000008BF9F3
 :04008600000012F76D
+:040C400000000001AF
 :00000001FF
processing file asmFiles/ihit_dhit_same.asm (1 of 1)
--- memsim.hex	2021-10-24 04:07:28.835339363 -0400
+++ memcpu.hex	2021-10-24 04:07:28.745336105 -0400
@@ -6,4 +6,5 @@
 :04000500AC43000800
 :040006001403FFFDE3
 :04000700FFFFFFFFF9
+:04003E0000000003BB
 :00000001FF
processing file asmFiles/ihit_dhit_same.asm (1 of 1)
--- memsim.hex	2021-10-24 04:09:01.125679893 -0400
+++ memcpu.hex	2021-10-24 04:09:01.005675549 -0400
@@ -6,5 +6,5 @@
 :04000500AC43000800
 :040006001403FFFDE3
 :04000700FFFFFFFFF9
-:040C400000000003AD
+:04003E0000000003BB
 :00000001FF
processing file asmFiles/beq_not_taken.asm (1 of 2)
processing file asmFiles/beq_taken.asm (2 of 2)
--- memsim.hex	2021-10-24 04:09:13.156115345 -0400
+++ memcpu.hex	2021-10-24 04:09:13.051111544 -0400
@@ -9,5 +9,4 @@
 :04000800ACA200089E
 :040009001044FFFDA3
 :04000A001062FFFA87
-:043FC20000000009F2
 :00000001FF
processing file asmFiles/RAW.asm (1 of 1)
processing file asmFiles/ihit_dhit_same.asm (1 of 1)
--- memsim.hex	2021-10-24 04:14:31.907652835 -0400
+++ memcpu.hex	2021-10-24 04:14:31.809649288 -0400
@@ -6,5 +6,5 @@
 :04000500AC43000800
 :040006001403FFFDE3
 :04000700FFFFFFFFF9
-:040C400000000003AD
+:04003E0000000003BB
 :00000001FF
processing file asmFiles/RAW.asm (1 of 1)
processing file asmFiles/ihit_dhit_same.asm (1 of 1)
--- memsim.hex	2021-10-24 04:14:47.144204335 -0400
+++ memcpu.hex	2021-10-24 04:14:47.053201042 -0400
@@ -6,4 +6,5 @@
 :04000500AC43000800
 :040006001403FFFDE3
 :04000700FFFFFFFFF9
+:04003E0000000003BB
 :00000001FF
